builtin/aws/lambda/function_url/metadata.hcl

```diff
@@ -0,0 +1,11 @@
+integration {
+  name        = "Lambda Function URL"
+  description = "TODO"
```

Comment:
@hashicorp/waypoint-ecosystem, Small callout here that there's still a TODO here as well